core/src/test/scala/unit/kafka/utils/ReplicationUtilsTest.scala

```diff
@@ -17,7 +17,7 @@
 
 package kafka.utils
 
-import kafka.server.{KafkaConfig, AsyncReplicaFetcherManager, ReplicaManager}
+import kafka.server.{AsyncReplicaFetcherManager, KafkaConfig, ReplicaFetcherManager, ReplicaManager}
```

Comment:
is AsyncReplicaFetcherManager still used in this test?

Reply:
No. It's been removed.